sycl/source/detail/device_impl.cpp

```diff
@@ -510,6 +510,7 @@ EXPORT_GET_INFO(ext::intel::info::device::current_clock_throttle_reasons)
 EXPORT_GET_INFO(ext::intel::info::device::fan_speed)
 EXPORT_GET_INFO(ext::intel::info::device::min_power_limit)
 EXPORT_GET_INFO(ext::intel::info::device::max_power_limit)
+EXPORT_GET_INFO(ext::intel::info::device::luid)
```

Comment:
I first replied to a similar export below.

This one isn't needed either, I think.